src/tasks/processBulkUploadTask.ts

```diff
@@ -1,44 +1,53 @@
+/* eslint-disable max-lines -- This is a big file but that's the nature of the beast for now
```

Comment:
I'm not sure whether to protest this or not.

Reply:
If it helps: I do protest it -- but it's not solvable without adding weeks of development IMHO (which is why I captured the need in the referenced issue)